### DISCUSSION TITLE
I keep getting this error message in terminal, I hope I am safe? I am scared  it will crash my PC. What can be the Issue please.

### DISCUSSION THREAD

**Discussion by Mthought99** [nextauthjs/next-auth#11198] [Help]
Compiled /_not-found in 9.5s (594 modules)
GET /api/auth/providers 404 in 1162ms
 POST /api/auth/_log 404 in 627ms
 GET /api/auth/providers 404 in 117ms
 POST /api/auth/_log 404 in 105ms
 GET /api/auth/providers 404 in 102ms
 POST /api/auth/_log 404 in 109ms
 GET /api/auth/providers 404 in 110ms
 POST /api/auth/_log 404 in 124ms
 GET /api/auth/providers 404 in 142ms
 POST /api/auth/_log 404 in 108ms
 GET /api/auth/providers 404 in 94ms
 POST /api/auth/_log 404 in 110ms
 GET /api/auth/providers 404 in 114ms
 POST /api/auth/_log 404 in 75ms
 GET /api/auth/providers 404 in 100ms
 GET /api/auth/providers 404 in 97ms
 POST /api/auth/_log 404 in 120ms
 POST /api/auth/_log 404 in 131ms
 GET /api/auth/providers 404 in 133ms
 POST /api/auth/_log 404 in 89ms
 GET /api/auth/providers 404 in 243ms
 POST /api/auth/_log 404 in 301ms
 GET /api/auth/providers 404 in 334ms
 POST /api/auth/_log 404 in 195ms
 GET /api/auth/providers 404 in 207ms
 POST /api/auth/_log 404 in 151ms
 GET /api/auth/providers 404 in 156ms
 POST /api/auth/_log 404 in 150ms
 GET /api/auth/providers 404 in 153ms
 POST /api/auth/_log 404 in 120ms
 GET /api/auth/providers 404 in 119ms
 POST /api/auth/_log 404 in 110ms
 GET /api/auth/providers 404 in 113ms
 POST /api/auth/_log 404 in 124ms
 GET /api/auth/providers 404 in 130ms
 POST /api/auth/_log 404 in 121ms
 GET /api/auth/providers 404 in 143ms
 POST /api/auth/_log 404 in 151ms
 GET /api/auth/providers 404 in 158ms
 GET /api/auth/providers 404 in 149ms
 POST /api/auth/_log 404 in 144ms
 POST /api/auth/_log 404 in 147ms
 GET /api/auth/providers 404 in 148ms
 POST /api/auth/_log 404 in 81ms
 GET /api/auth/providers 404 in 78ms
 GET /api/auth/providers 404 in 84ms
 POST /api/auth/_log 404 in 84ms
 GET /api/auth/providers 404 in 61ms
 POST /api/auth/_log 404 in 72ms
 GET /api/auth/providers 404 in 88ms
 POST /api/auth/_log 404 in 95ms
 POST /api/auth/_log 404 in 65ms
 GET /api/auth/providers 404 in 102ms
 GET /api/auth/providers 404 in 94ms
 POST /api/auth/_log 404 in 122ms
 POST /api/auth/_log 404 in 107ms
 GET /api/auth/providers 404 in 116ms
 POST /api/auth/_log 404 in 86ms
 GET /api/auth/providers 404 in 104ms
 POST /api/auth/_log 404 in 121ms
 GET /api/auth/providers 404 in 122ms
 GET /api/auth/providers 404 in 125ms
 POST /api/auth/_log 404 in 129ms
 POST /api/auth/_log 404 in 82ms
 GET /api/auth/providers 404 in 73ms
 POST /api/auth/_log 404 in 101ms
 GET /api/auth/providers 404 in 95ms
 POST /api/auth/_log 404 in 55ms
 GET /api/auth/providers 404 in 208ms
 POST /api/auth/_log 404 in 219ms
 GET /api/auth/providers 404 in 221ms
 POST /api/auth/_log 404 in 84ms
 GET /api/auth/providers 404 in 98ms
 POST /api/auth/_log 404 in 89ms
 GET /api/auth/providers 404 in 88ms
 GET /api/auth/providers 404 in 118ms
 POST /api/auth/_log 404 in 121ms
 POST /api/auth/_log 404 in 73ms
 GET /api/auth/providers 404 in 78ms
 POST /api/auth/_log 404 in 85ms
 GET /api/auth/providers 404 in 85ms
 POST /api/auth/_log 404 in 69ms
 GET /api/auth/providers 404 in 84ms
 POST /api/auth/_log 404 in 100ms
 GET /api/auth/providers 404 in 101ms
 POST /api/auth/_log 404 in 84ms
 GET /api/auth/providers 404 in 84ms
 POST /api/auth/_log 404 in 76ms
 GET /api/auth/providers 404 in 88ms
 POST /api/auth/_log 404 in 89ms
 GET /api/auth/providers 404 in 93ms
 POST /api/auth/_log 404 in 103ms
 GET /api/auth/providers 404 in 107ms
 POST /api/auth/_log 404 in 89ms
 GET /api/auth/providers 404 in 91ms
 POST /api/auth/_log 404 in 73ms
 GET /api/auth/providers 404 in 85ms
 POST /api/auth/_log 404 in 103ms
 GET /api/auth/providers 404 in 103ms
 POST /api/auth/_log 404 in 84ms
 GET /api/auth/providers 404 in 108ms
 POST /api/auth/_log 404 in 82ms
 GET /api/auth/providers 404 in 82ms
 POST /api/auth/_log 404 in 80ms
 GET /api/auth/providers 404 in 86ms
 POST /api/auth/_log 404 in 82ms
 GET /api/auth/providers 404 in 86ms
 GET /api/auth/providers 404 in 83ms
 POST /api/auth/_log 404 in 86ms
 POST /api/auth/_log 404 in 79ms
 GET /api/auth/providers 404 in 90ms
 GET /api/auth/providers 404 in 88ms
 POST /api/auth/_log 404 in 97ms
 POST /api/auth/_log 404 in 85ms
 GET /api/auth/providers 404 in 93ms
 POST /api/auth/_log 404 in 53ms
 GET /api/auth/providers 404 in 54ms
 POST /api/auth/_log 404 in 75ms
 GET /api/auth/providers 404 in 78ms
 POST /api/auth/_log 404 in 83ms
 GET /api/auth/providers 404 in 79ms
 POST /api/auth/_log 404 in 74ms
 GET /api/auth/providers 404 in 75ms
 GET /api/auth/providers 404 in 108ms
 POST /api/auth/_log 404 in 113ms
 POST /api/auth/_log 404 in 82ms
 GET /api/auth/providers 404 in 88ms
 GET /api/auth/providers 404 in 93ms
 POST /api/auth/_log 404 in 108ms
 GET /api/auth/providers 404 in 81ms
 POST /api/auth/_log 404 in 84ms
 POST /api/auth/_log 404 in 101ms
 GET /api/auth/providers 404 in 106ms
 POST /api/auth/_log 404 in 79ms
 GET /api/auth/providers 404 in 73ms
 POST /api/auth/_log 404 in 86ms
 GET /api/auth/providers 404 in 89ms
 POST /api/auth/_log 404 in 72ms
 GET /api/auth/providers 404 in 88ms
 POST /api/auth/_log 404 in 62ms
 GET /api/auth/providers 404 in 53ms
 POST /api/auth/_log 404 in 93ms
 GET /api/auth/providers 404 in 94ms
^X POST /api/auth/_log 404 in 99ms
 GET /api/auth/providers 404 in 89ms
 POST /api/auth/_log 404 in 126ms
 GET /api/auth/providers 404 in 128ms
 POST /api/auth/_log 404 in 82ms
 GET /api/auth/providers 404 in 88ms
 POST /api/auth/_log 404 in 68ms
 GET /api/auth/providers 404 in 97ms
 POST /api/auth/_log 404 in 106ms
 GET /api/auth/providers 404 in 108ms
 GET /api/auth/providers 404 in 142ms
 POST /api/auth/_log 404 in 146ms
 GET /api/auth/providers 404 in 128ms
 POST /api/auth/_log 404 in 136ms
 POST /api/auth/_log 404 in 104ms
 GET /api/auth/providers 404 in 110ms
 POST /api/auth/_log 404 in 65ms
 GET /api/auth/providers 404 in 86ms
 POST /api/auth/_log 404 in 102ms
 GET /api/auth/providers 404 in 109ms
 POST /api/auth/_log 404 in 73ms
 GET /api/auth/providers 404 in 68ms
 POST /api/auth/_log 404 in 102ms
 GET /api/auth/providers 404 in 104ms
 POST /api/auth/_log 404 in 73ms
 GET /api/auth/providers 404 in 98ms
 POST /api/auth/_log 404 in 86ms
 GET /api/auth/providers 404 in 92ms
 POST /api/auth/_log 404 in 88ms
 GET /api/auth/providers 404 in 93ms

**Comment by aynaash** [comment]
I am getting the same issue I dont know why
